packages/integration-tests/test-projects/data-storage-auth-with-triggers-cjs/amplify/data/echo/handler.cjs

```diff
@@ -0,0 +1,7 @@
+/**
+ * Hello world lambda used for testing
+ */
+export const handler = async () => {
+  // eslint-disable-next-line no-console
```

Comment:
similar feedback here, I'd disable this in `test-projects` folder.

Reply:
I can just change these lambdas to return a dummy string

Reply:
btw. I wonder why this wouldn't cover that https://github.com/aws-amplify/amplify-backend/blob/e6fab217287b75b23f0c6492ab5114e83dd7b4db/packages/integration-tests/.eslintrc.json#L3 .

Reply:
Perhaps my editor didn't resolve the override correctly. I'm not sure I ever actually ran the linter on it, it was just underlined in the editor. Regardless, I've removed the console log lines here